app/views/organization_account_requests_mailer/create_new_organization_account_request.html.erb

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <p>Hi Dev Team,</p>
+
+    <p>We’ve got a new organization request! Here are the details:</p>
+
+    <ul>
+      <li><strong>Organization Name:</strong> <%= @organization_name %></li>
+      <li><strong>Contact Name:</strong> <%= @requester_name %></li>
+      <li><strong>Organization Phone Number:</strong> <%= @organization_phone %></li>
+      <li><strong>Organization Email:</strong> <%= @organization_email %></li>
+      # <li><strong>Country:</strong> <%= @country %></li>
+      # <li><strong>City/Town:</strong> <%= @city_town %></li>
+      # <li><strong>Province/State:</strong> <%= @province_state %></li>
```

Comment:
@kasugaijin @jmilljr24 @mononoken One thing to think about. The country and state/province information comes back as codes and not human readable names, which is good for saving records to the database, but for vetting would it be more helpful to have the names?

Reply:
Yes that would be nice. I think we can tackle that in a separate issue.